### ISSUE TITLE
[SQS] False positive for I3013 when creating a queue without an explicit retention period

### ISSUE THREAD

**Issue by AlexChesters** [aws-cloudformation/cfn-lint#4107]
### CloudFormation Lint Version

cfn-lint 1.33.2

### What operating system are you using?

Mac

### Describe the bug

The `MessageRetentionPeriod` is not a required parameter (https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-sqs-queue.html#cfn-sqs-queue-messageretentionperiod), but `cfn-lint` errors stating it is required.

### Expected behavior

cfn-lint should not indicate optional properties are required

### Reproduction template

```yaml
Resources:
  Queue:
    Type: AWS::SQS::Queue
```

**Comment by kddejong** [comment]
This is working as expected and as planned.  This is an informational level error that is asking you to specifically set this property so you are intentional of your data retention.

```
➜  cfn-lint --include-checks I -- local/issue/4107.yaml
I3011 'DeletionPolicy' is a required property (The default action when replacing/removing a resource is to delete it. Set explicit values for stateful resource)
local/issue/4107.yaml:2:3

I3011 'UpdateReplacePolicy' is a required property (The default action when replacing/removing a resource is to delete it. Set explicit values for stateful resource)
local/issue/4107.yaml:2:3

I3013 'MessageRetentionPeriod' is a required property (The default retention period will delete the data after a pre-defined time. Set an explicit values to avoid data loss on resource)
local/issue/4107.yaml:2:3
```

This will return nothing.
```
➜  cfn-lint -- local/issue/4107.yaml
```
